docs/source/community/persons_of_interest.rst

```diff
@@ -15,116 +15,136 @@ General Maintainers
 Module-level maintainers
 ------------------------
 
+torch.*
+~~~~~~~
+
+-  Greg Chanan (`gchanan <https://github.com/gchanan>`__)
+-  Soumith Chintala (`soumith <https://github.com/soumith>`__)
+-  [linear algebra] Vishwak Srinivasan (`vishwakftw <https://github.com/vishwakftw>`__)
+
+torch.nn
+~~~~~~~~
+
+-  Thomas Viehmann (`t-vi <https://github.com/t-vi>`__)
+-  Adam Paszke (`apaszke <https://github.com/apaszke>`__)
+-  Greg Chanan (`gchanan <https://github.com/gchanan>`__)
+-  Soumith Chintala (`soumith <https://github.com/soumith>`__)
+-  Sam Gross (`colesbury <https://github.com/colesbury>`__)
+
+torch.optim
+~~~~~~~~~~~
+
+-  Vincent Quenneville-Belair (`vincentqb <https://github.com/vincentqb>`__)
+-  Soumith Chintala (`soumith <https://github.com/soumith>`__)
+
+Autograd Engine
+~~~~~~~~~~~~~~~
+
+-  Edward Yang (`ezyang <https://github.com/ezyang>`__)
+-  Alban Desmaison (`alband <https://github.com/alband>`__)
+-  (sunsetting) Adam Paszke (`apaszke <https://github.com/apaszke>`__)
```

Comment:
FWIW I'm reading every single autograd PR, even if I don't comment on it, so I wouldn't agree with that 😬

Reply:
Ok, sure, it was @soumith's suggestion :-P

Reply:
Thanks @soumith 😊